src/index.js

```diff
@@ -39,6 +39,7 @@ class Onboarding extends Component {
     Animated.timing(this.state.backgroundColorAnim, {
       toValue: 1,
       duration: this.props.transitionAnimationDuration,
+      useNativeDriver: false,
```

Comment:
This is false because this Value is being used to animate background color which cannot be animated using Native Driver.
https://reactnative.dev/docs/animations#caveats

Reply:
Okay, so I won't merge? It's only a warning after all.

Reply:
Well i would prefer that you would merge.
I was just mentioning why one place `useNativeDriver` was true and another place it was false.

 Without this commit, the DoneButton 's animation would not be native driven.

And maybe it's a warning in v0.62 but it might change to an error in the next version onwards.

Reply:
Alright!